database/migrations/2017_12_13_081248_create_post_tags_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreatePostTagsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('post_tags', function (Blueprint $table) {
```

Comment:
Ở mối quan hệ nhiều nhiều đặt tên như vậy là sai rồi anh. Tên table có dạng a-z và số ít. Ví dụ: `post_tag'. Để hiểu rõ hơn a xem ở đây https://laravel.com/docs/5.5/eloquent-relationships#many-to-many